public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -100,6 +104,7 @@ export class TableContainer extends PureComponent<Props> {
             {frames.map((data, i) => (
               <PanelChrome
                 key={data.refId || `table-${i}`}
+                // @ts-expect-error expects a string while a ReactElement is valid
                 title={this.getTableTitle(dataFrames, data, i)}
```

Comment:
<img width="938" alt="image" src="https://github.com/grafana/grafana/assets/3773351/a9285807-2127-4e3c-a3e7-6a1e43662ef1">

To be honest, I couldn't find anything in the docs, but I believe this happens, because `title` is also a global HTML attribute.